lib/gringotts/gateways/cams.ex

```diff
@@ -87,14 +87,13 @@ defmodule Gringotts.Gateways.Cams do
     - Refund
   """
   @live_url  "https://secure.centralams.com/gw/api/transact.php"
-  @default_currency  "USD"
   @headers  [{"Content-Type", "application/x-www-form-urlencoded"}]
```

Comment:
Please mention that the unit of amount is not cents, in the "Scope of this module" section on line 52. See how it is done in MONEI docs.